Sources/Edit/ZLEditVideoViewController.swift

```diff
@@ -268,7 +268,7 @@ public class ZLEditVideoViewController: UIViewController {
     }
     
     @objc private func cancelBtnClick() {
-        dismiss(animated: animateDismiss, completion: nil)
+        dismiss(animated: animateDismiss) { self.editFinishBlock?(nil) }
```

Comment:
OK, compared to the image editor, the cancellation callback is missing. However, to better differentiate between no editing and editing being canceled for videos, e.g.,
```swift
// Max deviation is 0.01
if abs(d - round(CGFloat(avAsset.duration.seconds))) <= 0.01 {
    dismiss(animated: animateDismiss) {
        self.editFinishBlock?(nil)
}
    return
}
```
it’s best to add a `cancelEditBlock`. Please modify the code accordingly and resubmit it.
Thanks.